shared/ssa/qlpack.yml

```diff
@@ -0,0 +1,4 @@
+name: codeql/shared-ssa
+version: 0.0.1
```

Comment:
No need to make a decision for this PR, but we may want to make the version `0.0.1-dev` like the other packs and this prevents accidental publishing. Our release process would strip the `-dev` publish the version and bump the version after publishing.